### ISSUE TITLE
Use CodeBase for Gitlab Support

### ISSUE THREAD

**Issue by CallMeRoot** [Microsoft/vscode-github-issues-prs#20]
Hey there,
we are mainly using Gitlab and I would like to ask if you got any plans for adding other Services or if it would be ok to try and build this from a fork here?

Cheers,
Finch

**Comment by chrmarti** [comment]
No plans at the moment and it seems unlikely unless the API is the same as GitHub's. (Which I guess it is not.)

**Comment by CallMeRoot** [comment]
Yeah, just saw that. Just like the way all this is integrated into code... Maybe I'll try this week to "port" it.

**Comment by jens1o** [comment]
@hjfinch Have you did that? I'd like to avoid a vendor lock-in with Microsoft (acquired GitHub and npm and many other companies) etc.
